ui/src/ui/components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,89 @@
+// Import packages
+import React, { useState } from 'react';
+import toast from 'react-hot-toast';
+import { useNavigate } from 'react-router-dom';
+
+// Importing react icons
+import { AiOutlineClose, AiOutlineUser } from 'react-icons/ai';
+
+// Import components
+import SpecialButton from '../SpecialButton/SpecialButton';
+import InputField from '../InputField/InputField';
+
+// import rtk query
+import { useForgotPasswordMutation } from '../../../store/api';
+
+// Import css
+import './ForgotPassword.css';
+import '../../../App.css';
```

Comment:
App.css is already imported in the app.js component why do you need to import it again?

Reply:
Done